test/goldens/comments.dart.golden

```diff
@@ -6,54 +6,56 @@
 #^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
 >
 >/// Multiline dartdoc comment.
-#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.block.documentation.dart
+#^^^ comment.block.documentation.dart
+#    ^^^^^^^^^ support.class.dart
+#                             ^ punctuation.dot.dart
```

Comment:
These changes don't seem right - it seems like some of the comments are being parsed as code.

I'll see if I can repro this tomorrow and post back.

Reply:
I removed the extra commit again. Even though I then get local errors, the CI might now pass.